examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -228,7 +228,9 @@
     "### Save results  and perform evaluation\n",
     "After the model has predicted trajectories for our evaluation set, we can save them in a `csv` file. To simulate a complete evaluation session we can also save the ground truth in another `csv` and get the score.\n",
     "\n",
-    "We will get `future_num_frames` values, corresponding to the Displacement Error (mean of squared errors / l2-loss between predicted point and ground truth point) for that timestep."
+    "Pleae note that our metric support multi-modalities predictions (i.e. multiple predictions for a single GT trajectories, each one with its own confidence between 0 and 1)\n",
```

Comment:
Maybe add that these confidences should add up to 1? Could also be understood as if the confidences are independent between 0 and 1.

Reply:
Language: Please note that our metric supportS MULTI-MODAL predictions (i.e. multiple predictions for [REMOVE: a] single GT trajectories, each one with its own confidence between 0 and 1) [add .]